<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="testLoginSuite" duration-ms="4834" started-at="2019-04-27T13:26:54Z" finished-at="2019-04-27T13:26:59Z">
    <groups>
    </groups>
    <test name="loginTest" duration-ms="4834" started-at="2019-04-27T13:26:54Z" finished-at="2019-04-27T13:26:59Z">
      <class name="Jarvis.test.cases.LoginTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:Jarvis.test.cases.LoginTest@3498ed]" name="setup" is-config="true" duration-ms="3897" started-at="2019-04-27T13:26:54Z" finished-at="2019-04-27T13:26:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:Jarvis.test.cases.LoginTest@3498ed]" name="loginTest" duration-ms="1" started-at="2019-04-27T13:26:58Z" finished-at="2019-04-27T13:26:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[yangxr]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Jarvis.test.page.LoginPage.login(LoginPage.java:54)
at Jarvis.test.cases.LoginTest.loginTest(LoginTest.java:34)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:Jarvis.test.cases.LoginTest@3498ed]" name="teardown" is-config="true" duration-ms="921" started-at="2019-04-27T13:26:58Z" finished-at="2019-04-27T13:26:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Jarvis.test.cases.LoginTest -->
    </test> <!-- loginTest -->
  </suite> <!-- testLoginSuite -->
</testng-results>
